llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -953,8 +953,17 @@ class SelectionDAG {
   }
 
   /// Insert \p SubVec at the \p Idx element of \p Vec.
+  /// If \p SkipUndef is true and \p SubVec is UNDEF/POISON, then \p Vec is
+  /// returned.
```

Comment:
This is a strange pattern that I feel like should be avoided. I don't understand why this would need semantic treatment. These get* functions should just return with the node with the requested operands